app/admin/books.rb

```diff
@@ -0,0 +1,21 @@
+ActiveAdmin.register Book do
+  permit_params :title, :image
+
+  form do |f|
+    f.inputs "Books" do
+      f.input :title
+      f.input :image, :as => :file
```

Comment:
`:as => :file`は不要です。

```suggestion
      f.input :image
```

Reply:
- `, :as => :file` を削除しました。